### PR TITLE
Page content/styling fixes and css cleanup

### LINKED ISSUE #277: Privacy Page Visual Formatting
**Please describe what has to be done**

- [x] Markdown needs to be reformatted for proper displaying on the page. 
- [ ] Remove the "After developing our privacy policy..." note

### PR DESCRIPTION
**Pull request by annieshi6**
### Description of change(s), including why:

- privacy markdown reformatted
- update contact and about pages to use the new theme
- password placeholders standardized and password fields restyled bc current styling makes the difference between password placeholder and text jarring
- type="password" added to input fields where it was missing
- added another input field to accountinfo so changing password makes more sense (new password/confirm new/confirm old)
- misc css cleanup

### Type of change(s)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Any work to still do or other concerns:

- (none)

### Closing issues:

- Closes #277 #248 #247 #246

### CODE REVIEW

File: PollBuddy-Server/frontend/src/pages/Privacy/Privacy.md [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
 
+Personally identifiable data, such as name and email, are collected via user input. Cookies may be used to keep track of the user’s session. Poll Buddy may also collect IP addresses and browser information for anti-cheating functionality. This data is used only for Poll Buddy related functions, and none of the information is sold or shared with anyone else.
+Poll Buddy’s main function is to collect user answers to questions, and will only collect as much information is necessary to provide the site’s functionality. Information inputted by the user is stored potentially indefinitely, but you do have the option to request timely deletion of your data.
 
-After developing our privacy policy, it would be cool to write it all in a google doc/markdown/whatever
-file
-and then just display it on this page. I'm not sure how plausible that would be, but editing would be
-much easier.
-Regular text on this .js file is very difficult to update and maintain.
+If you have any remaining concerns regarding Poll Buddy’s privacy policy or requests regarding your data, please contact us at privacy@pollbuddy.app.
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/pages/Privacy/Privacy.md:16]: Can you also make the email a mailto link?

**Review by hbh7 — CHANGES_REQUESTED** [review]
Looks good! Just a few small things I'd like to see. I made comments on the relevant lines.

File: PollBuddy-Server/frontend/src/pages/AccountInfo/AccountInfo.js [hunk 2]
```diff
@@ -16,39 +16,18 @@ export default class AccountInfo extends Component {
             <label htmlFor="nameText">Name:</label>
             <input placeholder="SIS Man" className="form-control textBox" id="nameText"/>
             <label htmlFor="emailText">Email:</label>
-            <input placeholder="mans@rpi.edu" className="form-control textBox" id="emailText"/>
-            <label htmlFor="currentPasswordText">Current password:</label>
-            <input placeholder="shir1ey-is-my-gir1y" className="form-control textBox" id="currentPasswordText"/>
-            <label htmlFor="confirmPassworkText">Confirm password:</label>
-            <input placeholder="shir1ey-is-my-gir1y" className="form-control textBox" id="confirmPassworkText"/>
+            <input placeholder="sisman@rpi.edu" className="form-control textBox" id="emailText"/>
+            <label htmlFor="newPasswordText">New password:</label>
```
[the remark this answers is not in the document]
Comment (annieshi6) [reply PollBuddy-Server/frontend/src/pages/AccountInfo/AccountInfo.js:20]: hmm I'm not really sure how else to word it without literally putting "(Not required)" lol but looks like Neha's pr includes a separation between info and change password. Maybe these fields (old pw, new pw, confirm new pw) can go on a new page or only be made visible when you click change password?

[the remark this answers is not in the document]
Comment (hbh7) [reply PollBuddy-Server/frontend/src/pages/AccountInfo/AccountInfo.js:20]: Yep looks like that's taken care of

**Review by hbh7 — APPROVED** [review]
Looks good!

**Review by nbezanis — APPROVED** [review]

**Pull request merged**